istioctl/pkg/multicluster/remote_secret.go

```diff
@@ -72,6 +68,11 @@ const (
 	DefaultServiceAccountName = "istio-reader-service-account"
```

Comment:
Let's pull this out and make it a constant under `pkg/config/constants`

Reply:
done